src/builtin/echo.c

```diff
@@ -0,0 +1,51 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   echo.c                                             :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mhoyer <mhoyer@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/07/06 07:37:23 by mhoyer            #+#    #+#             */
+/*   Updated: 2023/07/06 07:37:29 by mhoyer           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdio.h>
```

Comment:
```suggestion
#include "builtin.h"
```
et dans builtin.h tu ajoute 
```c
#include "libft.h"
```
si t'en as besoin, ensuite tu peux rajouter le prototype de echo egalement